monzo.js

```diff
@@ -1,20 +1,83 @@
 var PD = require("probability-distributions");
 
-var goal = 25000;
+var goal = 2500000;
 var pledgeValues = [10,20,50,100,250,500,1000];
 var maxPledgeValue = pledgeValues[pledgeValues.length - 1];
-var dataset = PD.sample(pledgeValues, 1000, true, [7,6,5,4,3,2,1]);
+var dataset = PD.sample(pledgeValues, 100000, true, [7,6,5,4,3,2,1]);
```

Comment:
this line fails with a sample set size of 1000. The code should work regardless of the number of applicants